containers/cpp/.devcontainer/base.Dockerfile

```diff
@@ -8,6 +8,20 @@ RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
     && apt-get -y install build-essential cmake cppcheck valgrind clang lldb llvm gdb \
```

Comment:
Would it make sense not to include this `cmake` to avoid removing in order to add the one from vcpkg in case is asked?

Reply:
This should stay as-is to ensure we do not change the underlying dependencies of the core cpp base image as it exists today.